rename `Query` to `Statement` (#1250)
scylladb/scylla-rust-driver: docs/source/statements/usekeyspace.md

This method has a slightly worse latency than `Session::use_keyspace` - there are two roundtrips needed instead of one.
Therefore, `Session::use_keyspace` is the preferred method for setting keyspaces.

### Multiple use keyspace requests at once
Don't run multiple `use_keyspace` requests at once. 

> This could end up with half of connections using one keyspace and the other half using the other.

Reviewer: :camping:

Reviewer: ```suggestion
### Multiple use keyspace requests at once
Don't run multiple `use_keyspace` requests at once. 
This could end up with a part of connections using one keyspace and another part using another.
```
